src/main/webapp/scheduled-interviews.jsp

```diff
@@ -0,0 +1,30 @@
+<%@page import="java.util.ArrayList"%>
+<%@page import="java.util.List"%>
+<%@ page import="com.google.sps.data.ScheduledInterviewRequest" %>
+<%
+  List<ScheduledInterviewRequest> scheduledInterviews = (List<ScheduledInterviewRequest>)request.getAttribute("scheduledInterviews");
+  pageContext.setAttribute("scheduledInterviews", scheduledInterviews);
+%>
+<%@ taglib uri="http://java.sun.com/jsp/jstl/core" prefix="c" %>
+
+<c:choose>
+  <c:when test= "${empty scheduledInterviews}">
+    <h2 style="text-align: center">No Scheduled Interviews</h2>
+  </c:when>
+  <c:otherwise>
+    <c:forEach items= "${pageScope.scheduledInterviews}" var="scheduledInterview">
+      <div class="row">
+        <div class="card w-75 scheduled-interview-card">
```

Comment:
It might be worth-while to include the scheduled interview id somewhere (in a data-id field maybe) for easy access later on.

Reply:
Done